3회차/백동원/0_최빈수구하기.py

```diff
@@ -1,3 +1,17 @@
 import sys
 
 sys.stdin = open("_최빈수구하기.txt")
+
+T = int(input())
+for _ in range(T):
+    test_case = int(input())
+    score_amnt = []
+    index_list = []
+    score_list = list(map(int, input().split()))
+    score_kind = list(set(score_list))
+    for A in score_kind:
+        score_amnt.append(score_list.count(A))
+    for B in range(len(score_amnt)):
+        if score_amnt[B] == max(score_amnt):
```

Comment:
max 값이 for 문 탐색 중 크기가 변하지 않아요! max() 연산을 한 번만 하도록 수정해주시면 좋습니다
```suggestion
    m = max(score_amnt)
    for B in range(len(score_amnt)):
        if score_amnt[B] == m:
```

Reply:
아아 max 연산이 for 문 내에서 계속 되네요.
좋은 지적 감사합니다!